### DISCUSSION TITLE
Candidatura di: Sofia Fulgido

### DISCUSSION THREAD

**Discussion by Cadienvan** [Il-Libro-Open-Source/book#116] [Candidature Ambassadors In Corso]
Questo sondaggio viene aperto per tutto il mese di Settembre per indicare la preferenza o l'opposizione sulla candidatura a ruolo di Ambassador per:

[Sofia Fulgido](https://github.com/fulgido)

In caso di disaccordo, si protegge l'anonimato ma sarebbe gradita una spiegazione tramite commento a questo sondaggio con delle motivazioni.

Attualmente, vista la fase embrionale del progetto, non ci saranno limiti minimi di votazione e la semplice superiorità dei voti a favore sarà considerata sufficiente a procedere con la nomina.

**Comment by aogier** [comment]
che senso ha proporre una candidatura in un sondaggio che apri il 21 per chiudere il 30? Se non è possibile farlo durare un mese da qualsiasi data d'apertura non avrebbe comunque più senso aprire il sondaggio il primo del mese successivo?

**Reply by Cadienvan** [reply]
Ciao Alessandro,
i motivi sono presto detti: Siamo arrivati ad un numero veramente alto di Ambassador, numero inaspettato, e si è ritenuto necessario chiudere la possibilità di candidarsi, ma non sarebbe stato corretto togliere la possibilità agli eventuali "ritardatari", si fa per dire, di partecipare. Questo però alla condizione principale di riuscire a raccogliere un numero sufficiente di voti (Che ricordo essere "superiorità numerica rispetto ai negativi" al momento), chiaramente questo comporta che candidarsi il 30 da una possibilità ben inferiore che candidarsi l'1, ma pur sempre meglio che trovarsi le porte chiuse :)

**Reply by aogier** [reply]
ah ecco. Grazie per la risposta, ciao

**Comment by eppak** [comment]
Ciao @fulgido, 
Ti confermiamo che la votazione si chiude oggi e che è positiva. Benvenuto tra gli Ambassador, ti chiediamo di aggiungere Michael su Telegram «at»Cadienvan che provvederà ad aggiungerti al gruppo organizzativo. 
Ti ricordiamo di prendere visione, oltre delle linee guida, anche del nostro codice di condotta che trovi https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md. 
Il repository sulla governance invece lo trovi a questo indirizzo dove sono presenti anche i gruppi di lavoro. 
Ti aspettiamo!

**Comment by eppak** [comment]
Ciao fulgido, ti ricordiamo che per confermare la tua partecipazione al team Ambassador, ti basta aggiungere Michael su Telegram «at» Cadienvan che provvederà ad aggiungerti al gruppo organizzativo.
Ti ricordiamo di prendere visione, oltre delle linee guida, anche del nostro codice di condotta che trovi https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md.
Ti aspettiamo!

P.S. hai ancora fino a Domenica 15 Ottobre per confermare la tua partecipazione, dopodiché la mancanza di comunicazione verrà considerata una mancata accettazione di questo invito.
